docs/docsite/rst/os_guide/index.rst

```diff
@@ -1,8 +1,8 @@
 .. _os_guide_index:
 
-################################
-Using Ansible on Windows and BSD
-################################
+#######################################
+Using Ansible on Windows, BSD, and z/OS
```

Comment:
Shouldn't it explicitly say z/OS UNIX here? Confusing z/OS with z/OS UNIX was one of the big issues in https://forum.ansible.com/t/40091 after all.